src/index.html

```diff
@@ -1,12 +1,126 @@
+                <path d="M6,9l4-5H2Z" />
+              </svg> -->
+            </a>
+
+            <ul class="f-header__dropdown">
+              <li><a href="#0" class="f-header__dropdown-link">Spaceship 1</a></li>
+              <li><a href="#0" class="f-header__dropdown-link">Spaceship 2</a></li>
+              <li><a href="#0" class="f-header__dropdown-link">Spaceship 3</a></li>
+              <li><a href="#0" class="f-header__dropdown-link">Spaceship 4</a></li>
+              <li><a href="#0" class="f-header__dropdown-link">Spaceship 5</a></li>
+              <li><a href="#0" class="f-header__dropdown-link">Spaceship 6</a></li>
+            </ul>
+          </li>
+          <li class="f-header__item"><a href="#0" class="f-header__link">Planets</a></li>
+          <li class="f-header__item"><a href="#0" class="f-header__link">Races</a></li>
+        </ul>
+      </div>
+    </div>
+  </header>
+
+  <main class="container max-width-md">
+    <button class="btn btn--primary" aria-controls="modal-name-1">Show Comment</button>
+  </main>
+
+  <div class="modal modal--animate-scale flex flex-center bg-contrast-higher bg-opacity-90% padding-md js-modal"
+    id="modal-name-1">
+    <div class="modal__content width-100% max-width-xs max-height-100% overflow-auto bg radius-md shadow-md"
+      role="alertdialog" aria-labelledby="modal-title-1" aria-describedby="modal-description-1">
+      <header class="bg-contrast-lower padding-y-sm padding-x-md flex items-center justify-between">
+        <h4 class="text-truncate" id="modal-title-1">Movie title here</h4>
+
+        <button class="reset modal__close-btn modal__close-btn--inner hide@md js-modal__close js-tab-focus">
+          <svg class="icon" viewBox="0 0 20 20">
+            <title>Close modal window</title>
+            <g fill="none" stroke="currentColor" stroke-miterlimit="10" stroke-width="2" stroke-linecap="round"
+              stroke-linejoin="round">
+              <line x1="3" y1="3" x2="17" y2="17" />
+              <line x1="17" y1="3" x2="3" y2="17" />
+            </g>
+          </svg>
+        </button>
+      </header>
+
+      <div class="padding-y-sm padding-x-md">
+        <img src="https://upload.wikimedia.org/wikipedia/en/thumb/4/4e/Wonder_Woman_1984.png/220px-Wonder_Woman_1984.png" alt="" class="width-100%">
+        <div class="text-component">
+          <div class="padding-top-xs text-center">
+            <h3>This Movie Title</h3>
+          </div>
+          <p id="modal-description-1">Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae culpa, inventore
+            alias ab atque similique quod ea reprehenderit.</p>
+        </div>
+      </div>
+
+      <!-- <footer class="padding-md">
+        <div class="flex justify-end gap-xs">
+          <button class="btn btn--subtle js-modal__close">Cancel</button>
+          <button class="btn btn--primary">Install</button>
+        </div>
+      </footer> -->
```

Comment:
Can you just delete the commented parts from the file.

Reply:
Okay